docs/satellite-clone.md

```diff
@@ -23,6 +23,14 @@ After cloning:
 
 RHEL6 to RHEL7 migration:
   - You can clone RHEL 6 backup data to a RHEL 7 machine.  In this case, you must update the variable `rhel_migration` to `true` as explained later in this document.
+  - Typical Migration workflow:
```

Comment:
This is good to document the production clone workflow, but this isn't exclusive to a RHEL migration. We can put this into a separate section called "Production clone workflow" or similar